Makefile

```
@@ -29,4 +29,7 @@ run:
 constd:
 	go build -o .bin/constd -tags netgo -ldflags="-extldflags=-static" ./constd
 
+constd/dev:
```

```suggestion
constd/dev: constd
```

I ran into this trying to test things out. I didn't have `constd` installed already.